src/components/ConnectedTile.jsx

```diff
@@ -0,0 +1,45 @@
+import styles from '../styles/connectedTile'
+
+import React from 'react'
+import { connect } from 'react-redux'
+import { translate } from 'cozy-ui/react/I18n'
+import { CONNECTION_STATUS } from '../lib/CollectStore'
+import { getAccountName } from '../lib/helpers'
+
+import { getConnectionStatusForTrigger } from '../ducks/connections'
+
+import KonnectorTile from './KonnectorTile'
+
+const ConnectedTile = props => <KonnectorTile {...props} />
+
+const svgIcon = name => (
+  <svg className="item-status-icon">
+    <use xlinkHref={require(`../assets/sprites/icon-${name}.svg`)} /> }
+  </svg>
+)
+
+const stateIcon = status => {
+  switch (status) {
+    case CONNECTION_STATUS.ERRORED:
+      return svgIcon('warning')
+    case CONNECTION_STATUS.CONNECTED:
+      return svgIcon('check')
+    case CONNECTION_STATUS.RUNNING:
+      return <span className={styles['col-konnector-status-running']} />
+    default:
+      return null
+  }
+}
+
+const mapStateToProps = (state, props) => {
+  const { account } = props
+  return {
+    icon: stateIcon(
+      getConnectionStatusForTrigger(state.connections, props.trigger)
+    ),
+    subtitle: getAccountName(account),
+    route: `${props.konnector.slug}/${account._id}`
```

Comment:
I wonder if `subtitle` and `route` should be somewhere else than in the `mapStateToProps`, since they don't use the `state` at all. Maybe in the `ConnectedTile`?

Reply:
You're right, I'll move them.